data/json/materials.json

```diff
@@ -2226,7 +2246,13 @@
     "bash_dmg_verb": "squished",
     "cut_dmg_verb": "cut",
     "vitamins": [ [ "iron", 0.1 ] ],
-    "burn_data": [ { "fuel": 1, "smoke": 2, "burn": 1 }, { "fuel": 1, "smoke": 2, "burn": 2 }, { "fuel": 1, "smoke": 1, "burn": 3 } ],
+    "//": "This burn data assumes wheat and chaff. In this form, it burns pretty well.",
```

Comment:
The wheat material is primarily used for food, be it toast ems or spaghetti.

Reply:
Thanks for checking this one. Now that I think of it, harvested wheat would come with piles of straw as byproducts too. So we can safely drop this one down to essentially 0.